src/test/scala/no/ndla/articleapi/service/ConverterServiceTest.scala

```diff
@@ -273,46 +283,13 @@ class ConverterServiceTest extends UnitSuite with TestEnvironment {
   }
 
   test("toApiArticle converts a domain.Article to an api.Article") {
```

Comment:
Mangler tester for de nye konverteringsfunksjonene. Det som er viktig er å teste logikken rundt språkhåndteringen. F.ex at konverteringsfunksjonene skal returnere `None` dersom språket ikke finnes